app/javascript/store/modules/profile/actions.js

```diff
@@ -40,4 +44,20 @@ export default {
         commit(RECOMMENDATIONS_FETCH_ERROR, error);
       });
   },
+
+  [COMPUTE_PROFILE_PR_INFORMATION]({ commit }, {githubUserLogin, monthLimit}) {
```

Comment:
A space is required after '{'                        object-curly-spacing<br>A space is required before '}'                       object-curly-spacing